homework_3/task_1,1.py

```diff
@@ -0,0 +1,7 @@
+# Задайте список из нескольких чисел. Напишите программу, которая найдёт сумму элементов списка, стоящих на нечётной позиции.
+array = [2,3,1,3,5,10]
+print(array)
+sum = 0
+for i in array[1:len(array):2]:
+    sum+=i
+print(sum)
```

Comment:
```suggestion
array = [2, 3, 1, 3, 5, 10]
print(array)
sum = 0
for i in array[1:len(array):2]:
    sum += i
print(sum)
```
Следите, пожалуйста за пробелами.